src/jaxsim/math/rotation.py

```diff
@@ -79,9 +79,11 @@ def theta_is_not_zero(theta_and_v: Tuple[jtp.Float, jtp.Vector]) -> jtp.Matrix:
 
             return R.transpose()
 
-        return jax.lax.cond(
-            pred=(theta == 0.0),
-            true_fun=lambda operand: jnp.eye(3),
-            false_fun=theta_is_not_zero,
-            operand=(theta, vector),
+        return jaxlie.SO3.from_matrix(
+            jax.lax.cond(
+                pred=(theta == 0.0),
+                true_fun=lambda operand: jnp.eye(3),
+                false_fun=theta_is_not_zero,
+                operand=(theta, vector),
+            )
         )
```

Comment:
Ok just got it, this should not be a `jaxlie.SO3` object but a jax numpy vector. This `math.rotation` helper should not accept nor return any `jaxlie` type.

Reply:
In https://github.com/ami-iit/jaxsim/pull/212/commits/2e019dd3d6175b35e96a6d12e7b823c4ebf75dc6 I added `.as_matrix()`, so the return type is a `jax.numpy.array`